backend/package-lock.json

```diff
@@ -0,0 +1,1469 @@
+{
```

Comment:
We shouldn't have package-lock.json in backend folder

Reply:
Got it, will work on it

Reply:
Hi, @v0ldemar01  
since you are already here, could you please have a look at what Farid requested before? I think I fixed it